src/core/control/Control.cpp

```diff
@@ -2241,7 +2239,7 @@ auto Control::openFile(fs::path filepath, int scrollToPage, bool forceOpen) -> b
         switch (res) {
             case USE_PROPOSED:
                 if (!proposedPdfFilepath.empty()) {
-                    loadHandler.setPdfReplacement(proposedPdfFilepath, true);
+                    loadHandler.setPdfReplacement(proposedPdfFilepath, false);
```

Comment:
I have set this to false, since a pdf is now only proposed when not in attach mode